client/hosting/server-settings/components/sftp-card/style.scss

```diff
@@ -0,0 +1,26 @@
+.sftp-card__ssh-field .sftp-card__input {
+	margin-top: 5px;
+}
```

Comment:
Nit, I would move this selector closer to the other `.sftp-card__input` selector.